planeseg: False # Use URDF of the environment or planeseg visualisation.

world:
  urdf: "/home/thomas_cbrs/Desktop/edin/memmo_anymal/data/urdf/lab_scene.urdf" # Env URDF path
  heightmap: "/home/thomas_cbrs/Desktop/edin/memmo_anymal/data/lab_scene.dat"  # Heightmap path

params:
  margin: 0.06 # inner surface margin
  n_points: 6 # Maximum number of vertices for the surfaces
  method_id: 3 # Method to remove overlapping
  poly_size: 10 # Number of maximum point for convex decomposition.
  min_area: 0.03 # Minimum area to keep the surfaces
  N_phase: 3 # Number of step to proceed (--> N_phase * n_gait step in SL1M)
  N_phase_return: 2 # Number of step to return (N_phase_return surfaces for each foot)
  com: True # Optimisation of the CoM
  multiprocessing: False